doc/api/http.md

```diff
@@ -744,6 +744,39 @@ added: v0.4.0
 
 * {string} The request path.
 
+### `request.method`
+<!-- YAML
+added: REPLACEME
```

Comment:
Since this property already existed before this PR, this version needs to be set to the correct version when it was introduced.